charts/linkerd-base/values.yaml

```diff
@@ -0,0 +1,108 @@
+# Default values for linkerd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- control plane version. See Proxy section for proxy version
+linkerdVersion: linkerdVersionValue
+# -- enables the use of EndpointSlice informers for the destination service;
+# enableEndpointSlices should be set to true only if EndpointSlice K8s feature
+# gate is on; the feature is still experimental.
+enableEndpointSlices: false
```

Comment:
Do you think it would make more sense to have this in `linkerd-control-plane`? The flag enables it in the destination controller, mostly. If we treat it as a global value though, it might make more sense to have it here, so other deployments can leverage it. wdyt?

Reply:
Note that these charts are totally independent. We can't use values from one's values.yaml's into the other templates'.
`enableEndpointSlices` is actually one of the few cases where this value is used in both charts, and that's why it's declared in both values.yaml files. The user has to be careful to set the same value in both charts; it's the same thing we've have to do e.g. with `clusterDomain` across extensions.

Reply:
Thank you very much for the explanation! Makes sense.